components/ordhook-core/src/core/protocol/inscription_sequencing.rs

```diff
@@ -4,6 +4,7 @@ use std::{
     sync::Arc,
 };
 
+// TODO: add this in chainhook-sdk and import it here?
```

Comment:
I think importing directly from `bitcoin` in different components is OK, as long as they all use the same version (i.e. it's done via `workspace = true`). I think you should remove these TODOs